### ISSUE TITLE
Details component fails WCAG AA standard for colour ratio whilst on focus

### ISSUE THREAD

**Issue by paroxp** [alphagov/govuk-frontend#944]
## What

I've been addressed by a user, regarding the contrast in our application where we're using the [GOV.UK Frontend - Details](https://design-system.service.gov.uk/components/details/) component.

The colours are:

Background: `#ffbf47`
Text: `#005ea5`
Text on hover: `#2b8cc4`

Running these past [snook](https://snook.ca/technical/colour_contrast/colour.html#fg=005ea5,bg=ffbf47) or [accessible colors](http://accessible-colors.com/) service, will incline the colours are failing the `AA` standard with a `Your contrast ratio: 4.07` warning.

Proposal from the user:

> Change the text colour to black.

I imagine you'd need a designer to look at it or perhaps have a small chat among yourselves. But perhaps, using the purple of your palette (`#2e358b`), would work fine for you? It would avoid the use of `darken()` function or setting it to black.
